packages/rx_bloc_cli/README.md

```diff
@@ -50,9 +50,7 @@ Out of the box, a Rx Bloc CLI created projects includes:
 
 ✅ [Push notifications][push_notifications_lnk] - (Coming soon) Receive push notifications and messages to your device using Firebase Cloud Messaging
 
-✅ [Http client][dio_http_client_lnk] - Dio is an Http client, which supports Interceptors, Global configuration, FormData, Request Cancellation, File downloading, Timeout etc.
-
-✅ [Interceptors][interceptors_lnk] - Interceptors are a powerful mechanism that can monitor, rewrite, and retry calls.
+✅ [Http client][dio_http_client_lnk] - Dio is an Http client, which supports Interceptors[interceptors_lnk], Global configuration, FormData, Request Cancellation, File downloading, Timeout etc.
```

Comment:
There is a typo here: `...which supports Interceptors[interceptors_lnk]` should be `...which supports [Interceptors][interceptors_lnk]` so the link gets properly assigned to the marked word(s) before it.